src/components/screens/wallet/delegateProfile/delegateProfile.js

```diff
@@ -23,41 +23,45 @@ import PerformanceView from './performanceView';
 // };
 
 const DelegateProfile = ({
-  delegate, address, t, voters,
+  account, t, voters,
   // awaitingForgers, forgingTimes,
   lastBlockForged,
 }) => {
+  const { delegate } = account.dpos;
   useEffect(() => {
-    delegate.loadData();
     voters.loadData();
-  }, [address]);
+  }, [account]);
 
   useEffect(() => {
-    lastBlockForged.loadData({ height: delegate.data.dpos?.delegate?.lastForgedHeight });
-  }, [delegate.data.dpos?.delegate?.lastForgedHeight]);
+    lastBlockForged.loadData({ height: delegate?.lastForgedHeight });
```

Comment:
Here, we should not null check the `delegate` variable before making the request. Can you please move the null check before the api call? 

```suggestion
	if(delegate?.lastForgedHeight) {
		lastBlockForged.loadData({ height: delegate.lastForgedHeight });
	}
```